site/.performance_warning_ignore.json

```diff
@@ -26,5 +26,8 @@
   "/courses/<semester>/<course>/course_materials/upload",
   "/courses/<semester>/<course>/course_materials/delete",
   "/courses/<semester>/<course>/course_materials/modify_timestamp",
-  "/courses/<semester>/<course>/course_materials/edit"
+  "/courses/<semester>/<course>/course_materials/edit",
+  "/courses/<semester>/<course>/polls/newPoll",
+  "/courses/<semester>/<course>/polls/editPoll/submitEdits",
+  "/courses/<semester>/<course>/polls/deletePoll"
```

Comment:
Getting Doctrine to group insert operations into one statement seems to be impractical.  Given that these are all instructor-side routes, with only a handful of duplicated queries, these should not be problematic at all.